src/actions/actions_login.js

```diff
@@ -0,0 +1,45 @@
+import fetch from "cross-fetch";
+
+export const REQUEST_LOGIN = 'REQUEST_LOGIN';
```

Comment:
Let's add a comment for exported entities.

Reply:
Okay